385/Assets/Scripts/PlayerController.cs

```diff
@@ -128,10 +128,18 @@ public class PlayerController : MonoBehaviour
     public bool IsOnGround
         => onGround;
 
-	/// <summary>
-	/// Holds value of transform.position for convenience
-	/// </summary>
-	private Vector3 playerPos;
+	  /// <summary>
+	  /// Holds value of transform.position for convenience
+	  /// </summary>
+	  private Vector3 playerPos;
+
+    // array to store multiple collisions in case platforms are
+    // placed next to each other
+    private CollisionStorage[] collisions;

```

Comment:
I would instead recommend to use a `List<CollisionStorage>`, the `List` type is in my opinion much easier to work with.